src/Service/Program.cs

```diff
@@ -148,7 +154,23 @@ public static ILoggerFactory GetLoggerFactoryForLogLevel(LogLevel logLevel, Tele
                         )
                         .AddFilter<ApplicationInsightsLoggerProvider>(category: string.Empty, logLevel);
                     }
-
+#if NET8_0_OR_GREATER
+                    if(Startup.OpenTelemetryOptions.Enabled && !string.IsNullOrWhiteSpace(Startup.OpenTelemetryOptions.Endpoint))
+                    {
+                        builder.AddOpenTelemetry(logging =>
+                        {
+                            logging.IncludeFormattedMessage = true;
+                            logging.IncludeScopes = true;
+                            logging.SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(Startup.OpenTelemetryOptions.ServiceName!));
+                            logging.AddOtlpExporter(configure =>
```

Comment:
`timeoutMilliseconds`,  `BatchExportProcessorOptions` there are few other configs, how can a customer override those, is it intentional that these options are not provided?

Reply:
It is intentional since I didn't want the config for OTEL to become over-complicated. Every parameter sent by the user would then need to be translated to a cli value as well. We could leverage default values, but we might first want to decide what should be configurable and what can be a default value decided on code side. At the moment, I've decided to implement the fundamentals such as endpoint, headers, service-name and (following Aaron's review) protocol.